how-to/mail-services/install-exim4.md

```diff
@@ -17,21 +17,48 @@ To configure Exim4, run the following command:
 sudo dpkg-reconfigure exim4-config
 ```
 
-This displays a "wizard" user interface for configuring the software. For example, in Exim4 the configuration files are split amongst multiple files by default; if you wish to have them in one file you can configure this via the user interface.
+This displays a "wizard" user interface for configuring the software. One important question in this configuration is whether exim4 should use split configuration files, or a single configuration file.
 
-All configurable parameters from the user interface are stored in the `/etc/exim4/update-exim4.conf.conf` file. To re-configure the software you can either re-run the wizard, or manually edit this file using your preferred editor. 
+If using multiple configuration files, then the configuration will be split in a directory structure under `/etc/exim4/conf.d`, like so:
+```text
+/etc/exim4/
+└── conf.d
+    ├── acl
+    ├── auth
+    ├── main
+    ├── retry
+    ├── rewrite
+    ├── router
+    └── transport
+```
+Each subdirectory will contain one or more individual configuration files.
 
-Once you are finished, you can run the following command to generate the master configuration file:
-```bash
+If, however, exim4 was setup to use a single configuration file (which is the default), then that file will be `/etc/exim4/exim4.conf.template`. It will essentially be as if all individual configuration files from the previous layout were concatenated into a big single file.
+
+```{note}
+The default configuration layout for exim4 is the single configuration file one.
+```
+
+In any of these scenarios, after making a change to the configuration, the following command must be executed to update the actual configuration file that exim4 will use:
+
+```text
 sudo update-exim4.conf
 ```
 
-The master configuration file is stored in `/var/lib/exim4/config.autogenerated`.
+The `update-exim4.conf` command will update the master configuration file stored in `/var/lib/exim4/config.autogenerated`.
```

Comment:
```suggestion
The `update-exim4.conf` command updates the master configuration file stored in `/var/lib/exim4/config.autogenerated`.
```

Reply:
I changed the text to avoid using that term.